perf/benchmark/runner/runner.py

```diff
@@ -42,7 +42,10 @@ def pod_info(filterstr="", namespace=NAMESPACE, multi_ok=True):
     cmd = "kubectl -n {namespace} get pod {filterstr}  -o json".format(
         namespace=namespace, filterstr=filterstr)
     op = getoutput(cmd)
-    o = json.loads(op)
+    try:
+        o = json.loads(op)
```

Comment:
why is it failing? seems like we just ignore the error.. if it fails won't L49 fail?